broadcasters/mqtt.py

```diff
@@ -15,11 +15,24 @@ def start_processing_output(system_queue, mqtt_q):
     """
 
     def on_connect(client, userdata, flags, rc):
-        """Callback function executed upon the successful connection to the desired broker"""
+        """Callback function executed upon the successful connection to the desired broker
```

Comment:
This might be crazy nitpicky, so you can disregard it, but shouldn't the opening `"""` be on a line by itself? The closing `"""` is on its own line. I also think it looks better. 

I am still approving the PR, but it might be worth fixing before merging it.